### PR TITLE
[NIC-40] Add person editing

### PR DESCRIPTION
**Pull request by AksyonovaKaterina**

### CODE REVIEW

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultPersonService.kt [hunk 1]
```diff
@@ -73,4 +74,12 @@ class DefaultPersonService(
 
         return conversionService.convert(personRecord, PersonDto::class)
     }
+
+    override suspend fun edit(personId: String, personDto: PersonDto): PersonDto {
+        val personRecord = personComponent.get(ObjectId(personId))
+            .apply { merge(personDto) }
+            .let { personComponent.update(it) }
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultPersonService.kt]: I think you should use `conversionService` to convert `PersonDto` to `PersonEntity` and then use `personComponent` for updating `Person` via `merge` entity with entity.

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/PersonService.kt [hunk 2]
```diff
@@ -30,4 +30,6 @@ interface PersonService {
     ): List<PersonDto>
 
     suspend fun editContacts(personId: String, contactsDto: ContactsDto): PersonDto
+
+    suspend fun edit(personId: String, personDto: PersonDto): PersonDto
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/service/PersonService.kt]: maybe rename it to `update`?)

**Review by V7074 — CHANGES_REQUESTED** [review]

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/PersonService.kt (hunk 2 above)
Comment (AksyonovaKaterina) [reply backend/person-service/src/main/kotlin/com/nosql/personservice/service/PersonService.kt]: fixed

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultPersonService.kt (hunk 1 above)
Comment (AksyonovaKaterina) [reply backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultPersonService.kt]: fixed, and in contacts

**Review by V7074 — APPROVED** [review]

**Pull request merged**